examples/tests/sdk_tests/testing/assert.w

```diff
@@ -0,0 +1,62 @@
+bring util;
+bring testing;
+
```

Comment:
would prefer to alias as `bring testing as t;` but this doesn't work right now https://github.com/winglang/wing/issues/4032